tests/e2e/iks-cluster

```diff
@@ -214,8 +214,9 @@ function check_addon_pod_status {
   pod_name=$1
   while true; do
     attempts=$((attempts+1))
-    pod_status=$(kubectl get sts -n kube-system | grep "$pod_name"| awk '{print $2}')
-    if [   "$pod_status" = "1/1" ]; then
+    pod_status_sts=$(kubectl get sts -n kube-system | grep "$pod_name"| awk '{print $2}')
+    pod_status_deployment=$(kubectl get deployment -n kube-system | grep "$pod_name"| awk '{print $2}')
```

Comment:
Grep only for controller pods and node server pods directly and check status

Reply:
https://github.ibm.com/alchemy-containers/armada-storage/issues/4875 - created GHE to track this